dali/kernels/common/copy.h

```diff
@@ -52,6 +54,26 @@ void copy(const TensorView<StorageOut, TOut, NDimIn>& out,
   copy<StorageOut, StorageIn>(out.data, in.data, in.num_elements() * sizeof(TOut), stream);
 }
 
+
+/**
+ * Copies input TensorView and returns the output.
+ * @tparam DstAlloc Requested allocation type of the output TensorView.
+ *                  According to this parameter, StorageBackend of output TensorView will be determined
+ * @tparam NonconstT utility parameter, do not specify (leave default)
+ * @return The output consists of new TensorView along with pointer to its memory (as the TensorView doesn't own any)
+ */
+template<
+        AllocType DstAlloc, typename SrcBackend, typename T,
+        int ndims, typename NonconstT = typename std::remove_const<T>::type
```

Comment:
If you don't want anyone to specify this argument, than remove it. It's not needed, only the return type will be a bit longer.

Reply:
And all the occurrences :(

Reply:
@klecki I don't think anyone would ever do something like that. I'm wondering, however, how and if it impacts compilationo speed (it's another template parameter, after all...).
@szalpal You can add the following to include/dali/core/traits.h:
```
template <typename T>
using remove_const_t = typename std::remove_const<T>::type;

template <typename T>
using remove_cv_t = typename std::remove_cv<T>::type;
```
and use `remove_const_t<T>` instead of the tumorous growth `typename std::remove_const<T>::type`.